docs/devops-guide/quickstart.md

```diff
@@ -4,10 +4,10 @@ title: Self-Hosting Guide - Debian/Ubuntu server
 sidebar_label: Debian/Ubuntu server
 ---
 
-This document describes the steps for a quick Jitsi-Meet installation on a Debian-based GNU/Linux system.
+Follow these steps for a quick Jitsi-Meet installation on a Debian-based GNU/Linux system.
 The following distributions are supported out-of-the-box:
-- Debian 10 (Buster) or later
-- Ubuntu 18.04 (Bionic Beaver) or later
+- Debian 10 (Buster) or newer
```

Comment:
Debian 9 is also supported.

Reply:
Ok, I can change that.

Reply:
Update: Debian 9 is now EOL (see [here](https://www.debian.org/News/2020/20200718)), so I would not include it tbh.

Reply:
Well, I saw your other post about the LTS version (though in my oppinion these  give a false impression of security; I used debian for years and even more severe bugs were not fixed in most packages).
So I included Debian 9.